src/public/meeting.ts

```diff
@@ -127,6 +127,11 @@ export namespace meeting {
     MeetNow = 'MeetNow',
   }
 
+  export enum MeetingDetailsCallType {
```

Comment:
Why not just call this "CallType" to match the "MeetingType" syntax?

Reply:
CallType in the TMP codebase as used by the CMD team have different enums; they only differentiate between a one on one and conference call. A conference call in this case would include both the group call and a meeting. I wanted to avoid the potential confusion in the codebase, and named this enum something different. @ydogandjiev

Reply:
We should not design the developer facing external API surface based on implementation details of how Teams works internally. We should design it to be the best it can be for the app developer, and deal with things internally as needed.